pod/perlsub.pod

```diff
@@ -104,7 +104,7 @@ like a C<foreach> or a C<while>, the returned value is unspecified.  The
 empty sub returns the empty list.
 X<subroutine, return value> X<return value> X<return>
 
-Aside from an experimental facility (see L</Signatures> below),
+Aside from the "signatures" facility (see L</Signatures> below),
```

Comment:
Now that signatures are no longer experimental, would it not be better to write this in a positive sense, and move the `X<formal parameter>` and  `X<parameter, formal>` tags to the Signatures section?

Reply:
Good idea. Further rewordings pushed.
(I'll definitely have to remember to squash these all up before merge)